charts/rancher-csp-adapter/0.1.0+up0.1.0-rc5/templates/deployment.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Chart.Name }}
+  namespace: cattle-csp-adapter-system
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Chart.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Chart.Name }}
+    spec:
+      containers:
+      - env:

```

Comment:
Needs node selectors and tolerations for windows support

Needs service account that has permission to use a defined PSP + a default securityPolicy for hardened cluster support. Also needs additional job to patch default service account and add network policy, see internal Confluence doc on Helm Chart Requirements for more instructions / code that can be copied and pasted to support this.

Reply:
It's already using a non-default service account. We'll add hardened cluster and psp support separately here: https://github.com/rancher/rancher/issues/37841